pkg/odo/cli/experimental/experimental.go

```diff
@@ -0,0 +1,40 @@
+package experimental
+
+import (
+	"os"
+
+	"github.com/openshift/odo/pkg/config"
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/preference"
+)
+
+// IsExperimentalModeEnabled checks if the experimental mode has been enabled
+// via env variable or via odo preferences and returns boolean value.
+// By default experimental mode is disabled
+func IsExperimentalModeEnabled() bool {
```

Comment:
why is this function in the CLI package? It seems to have nothing to do with CLI. Can't we move it under pkg folder?

Reply:
Then I guess this can be moved under pkg/odo/util as we keep helper functions for CLI there. All our completion, validation helpers and cmdUtils are there.

Reply:
But, experimental mode is a feature and not just a helper. Would it make sense to add to the `utils` package?

Reply:
But it is accessible via the preference or the env variable, not as a stand alone command of odo , similar to the completion handlers. So in my opinion it's better to move it in the utils package.

Reply:
Ok. done!